feature/pom.xml

```diff
@@ -110,7 +110,6 @@ limitations under the License.
                 <feature>jclouds-compute</feature>
                 <feature>jclouds-api-atmos</feature>
                 <feature>jclouds-api-byon</feature>
-                <feature>jclouds-api-cloudsigma2</feature>
```

Comment:
Easier to pack all the features in one or several big super-features in `features.xml`. This way you only need to update `feature.xml` when adding/removing features.